.github/workflows/run-tests.yml

```
@@ -0,0 +1,33 @@
+name: Run Tests
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
```

```suggestion
  pull_request:
```

i think this will be enough